### DISCUSSION TITLE
Do instances cost money after being stopped?

### DISCUSSION THREAD

**Discussion by rew1nter** [pry0cc/axiom#757] [General]
I'm using AWS. Does it cost me money if I just shut the instances down and not remove them?

**Comment by pdelteil** [comment]
It's a good question, I'm not sure. 

Linode charges for every created instance, on and off still charges. With AWS it should be for every instance being on. 

@0xtavian Can you confirm ?

**Reply by rew1nter** [reply]
I found out D.O charges shutdown instances as well
